manifests/init.pp

```diff
@@ -339,6 +339,8 @@
     timeout => 0,
   }
 
+  $use_foreman_service = ! $passenger
```

Comment:
I get why we need to set passenger to false explicitly in the current setup, but I do find not setting `use_foreman_service` unobvious when looking at the parameters someone would be specifying. If we will reverse this very soon after the 1.22 release then I can live with it.

Reply:
Note this is currently not a parameter but rather a variable inside the body so the user can't override this. In the reverse proxy scenario I'm expanding this logic. Because I couldn't properly figure out an API, I opted not to expose this to the user directly.